parser/src/lexer.rs

```diff
@@ -74,8 +74,8 @@ pub enum LexicalErrorType {
 
 #[derive(Clone, Debug, Default, PartialEq, Serialize, Deserialize)]
 pub struct Location {
-    row: usize,
-    column: usize,
+    pub row: usize,
```

Comment:
These have been public before, but then made private with the accessor function `get_row` and `get_column`.

Reply:
Ah yeah, that's a leftover from the syn serde serializer.